src/packages/@ncigdc/modern_components/AffectedCasesTable/AffectedCasesTable.js

```diff
@@ -363,9 +363,9 @@ const Component = compose(
                     age_at_diagnosis: ageDisplay(diagnosis.age_at_diagnosis),
                     tumor_stage: diagnosis.tumor_stage,
                     days_to_last_follow_up: (diagnosis.days_to_last_follow_up ||
-                      0)
+                      '--')
```

Comment:
not sure but I don't think 0 makes sense in this context even if it did. what does 0 days since last visit mean in a diagnosis?